NOTES.UNIX

```diff
@@ -5,38 +5,86 @@
  For Unix/POSIX runtime systems on Windows, please see NOTES.WIN.
 
 
- Shared libraries and installation in non-standard locations
- -----------------------------------------------------------
-
- Binaries on Unix variants expect to find shared libraries in standard
- locations, such as /usr/lib, /usr/local/lib and some other locations
- configured in the system (for example /etc/ld.so.conf on some systems).
- If the libraries are installed in non-standard locations, binaries
- will not find them and therefore fail to run unless they get a bit of
- help from a defined RPATH or RUNPATH.  This can be applied by adding
- the appropriate linker flags to the configuration command, such as
- this (/usr/local/ssl was the default location for OpenSSL installation
- in versions before 1.1.0):
+ Using compiler to link programs and shared libraries
+ ----------------------------------------------------
 
-    $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
-        -Wl,-rpath,/usr/local/ssl/lib
+ OpenSSL's generated Makefile generally uses $(CC) (the C compiler) to
```

Comment:
Doesn't "generally" implies that there might be exclusion from what is being said? Well, we don't use cl to link on Windows, but **this** is NOTES.UNIX ;-) I suppose I should leave it to language review...

Reply:
Well, there is the LDCMD hack...  Do we want to mention that here?

Reply:
I don't see need to mention LDCMD here. It should be noted though that even LDCMD is expected to accept $(CC) flags and *not* linker ones. So that it's still not exclusion that can be implied by "generally". Maybe it would be appropriate to omit explicit "$(CC)" in favour of more abstract/general "C compiler command line"?

Reply:
On second thought if LDCMD is to be mentioned, then there is no other place than NOTES.UNIX. However, I wouldn't say that it should be mentioned in **this** paragraph. I mean it would be more appropriate to mention it in paragraph of its own, explaining the circumstances when it can be useful.

Reply:
That's a thought.  Meanwhile, I like "C compiler command line", so fixing